examples/local/conversational-query-example.py

```diff
@@ -1,30 +1,33 @@
 import mii
 
+from time import time
+
+
 # gpt2
 name = "microsoft/DialoGPT-medium"
 
 print(f"Querying {name}...")
 
-str = "DeepSpeed is the greatest"
+text = "DeepSpeed is the greatest"
 
 generator = mii.mii_query_handle(name + "_deployment")
 result = generator.query({
     'text': str,
-    'conversation_id': 3,
+    'conversation_id': time(),
```

Comment:
I think the better solution would be to:
- use a uuid instead of current time
- allow this to be an optional parameter ([hugginface.Conversation](https://huggingface.co/transformers/v4.11.3/main_classes/pipelines.html#transformers.Conversation) will auto-generate a uuid if no value is provided)

@jeffra thoughts?

Reply:
Good point, will update the code to use `UUID4` 👌

Reply:
About making `'conversation_id'` as an optional argument. I would rather do it in another PR as it's not "example" related but rather "core" related.

Reply:
> @mrwyattii It seems that with the current code implementation, `conversation_id` can only be an `int`.
> 
> Otherwise it will encounter the following error message : `TypeError: 'UUID' object cannot be interpreted as an integer`
> 
> Raise by : `mii/server_client.py", line 291, in _request_async_response`
> 
> It don't think that the desired behaviour as you stated, [transformers.Conversation](https://huggingface.co/transformers/v4.11.3/main_classes/pipelines.html#transformers.Conversation) expect `conversation_id` to be either type of `uuid.UUID` or `None`.
> 
> I suggest to keep forward with the `time` option for the moment until being able to provide an `UUID` type. @mrwyattii is it fine to you?
> 
> Side note : Actually you can give it anything as argument it will still works fine.
> 
> i.e :
> 
> ```python
> conversation = Conversation(
>     text="Going to the movies tonight - any suggestions?",
>     conversation_id=lambda X : X # <-- Here
> )
> 
> from transformers import Conversation, pipeline
> 
> conversational_pipeline = pipeline("conversational")
> 
> result = conversational_pipeline([conversation])
> ```

I think the problem is in our prototext, where we expect the value passed in to be an `int`. This clearly doesn't align with `transformers.Conversation`. @jeffra and I were discussing refactoring the inputs for this particular task so that they are passed like other kwargs to `Conversation`. This will probably be in a separate PR. @jeffra how do you feel about using `time()` or perhaps defining a separate variable `conv_id = 3` to demonstrate that this is not a magic number in the example?

Reply:
Using `conv_id = 3` would be better than using time (my2c).

I would just suggest setting `conv_id` as `0`, again to demonstrate that `3` is not a magic number.

Reply:
> @jeffra and I were discussing refactoring the inputs for this particular task so that they are passed like other kwargs to Conversation

If I can help in any way, you be glad too ;)

Reply:
updated by [68d7d0e](https://github.com/microsoft/DeepSpeed-MII/pull/79/commits/68d7d0ef6a32c7a968aec8e1642983ffa9e09406)